src/components/useEventForm.js

```diff
@@ -1,16 +1,29 @@
 import { useState } from 'react'
 
-export default function useEventForm(addEvent) {
+export default function useEventForm(addEvent, updateEvent) {
   const [eventFormIsVisible, setEventFormIsVisible] = useState(false)
 
-  const showEventForm = () => setEventFormIsVisible(true)
+  const showEventForm = () => {
+    setEventFormIsVisible(true)
+  }
 
   const onEventSave = (event) => {
     addEvent(event)
     setEventFormIsVisible(false)
   }
+  const onEventSaveEdit = (event) => {
+    const newEvent = JSON.parse(JSON.stringify(event))
+    updateEvent(newEvent)
+    setEventFormIsVisible(false)
+  }
+  const goEventBack = () =>
+    window.location.reload() && setEventFormIsVisible(false)
```

Comment:
Do you need the logical operator && ?

Would this work too?
```
const goEventBack = () =>{
    window.location.reload()
    setEventFormIsVisible(false)}
```

Reply:
I need the logical operator, otherwise its not working.